src/lib/styles/global/colors.scss

```diff
@@ -40,30 +45,37 @@
   --cp-dark-900: #130a2c;
   --cp-dark-1000: rgb(var(--cp-dark-1000-rgb));
   --cp-dark-1000-rgb: 0, 0, 0;
+  // @deprecated (same as  --cp-dark-1000-20)
   --cp-dark-opaque: #00000020;
   --cp-dark-gradient-1: #0f0f4d;
   --cp-dark-gradient-2: #4d1259;
+  // @deprecated (replaced with --cp-dark-50--15% or --cp-dark-1000-20?)
```

Comment:
What is `--cp-dark-50--15%`? I couldn't find it.

Reply:
Just discussed this w/ the designer. We don't need it.